routing/grpcserver/server.go

```diff
@@ -0,0 +1,110 @@
+package grpcserver
+
+import (
+	"Dp218Go/protos"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"io"
+	"net/http"
+)
+
+type Client struct {
+	w    io.Writer
+	done chan struct{}
+}
+
+type Server struct {
+	client map[int]*Client
+	taken  map[int]bool
+	codes  map[int]int
+	in     chan *protos.ClientMessage
+	*protos.UnimplementedScooterServiceServer
+}
+
+
+func GISHandler(w http.ResponseWriter, r *http.Request) {
+	http.ServeFile(w, r, "./presentation/views/templates/html/index.html")
```

Comment:
Where do you actually use this file serving? We have only /templates folder now (without /presentation/view/)